x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -437,12 +446,12 @@ export class HeadlessChromiumDriver {
     // In order to get the inspector running, we have to know the page's internal ID (again, private)
     // in order to construct the final debugging URL.
 
+    const client = this.page._client();
     const target = this.page.target();
-    const client = await target.createCDPSession();
+    const targetId = target._targetId;
```

Comment:
Nit: this is unsupported code that seems could break with any upgrade. It had to be implemented this way in 7.6 for a bug with EMS maps tiles failing to download: https://github.com/elastic/kibana/pull/55137/files#diff-7eab27a89994a195c62fc9fb67dc9a93b9adc0fd86b22e6d260b0c8e7439acdbR64

Filed a bug to revisit that and attempt a more supported way to fix this: https://github.com/elastic/kibana/issues/140435